src/components/Charts/utils.ts

```diff
@@ -13,7 +13,7 @@ export const lastValidPoint = (data: Highcharts.Point[]) =>
 export const formatDecimal = (num: number, places = 2): string =>
   num.toFixed(places);
 
-export const formatPercent = (num: number, places = 0): string =>
+export const formatPercent = (num: number, places = 1): string =>
```

Comment:
Did you change this on purpose? Seems unrelated to the PR description. 😁

Reply:
Nope it was on accident.